README.md

```diff
@@ -1,2 +1,81 @@
 # apostrophe-svg-sprites
-SVG sprite elements as pieces
+
+`apostrophe-images` for SVGs in external sprite maps!
+![Managing sprites with apostrophe-svg-sprites](https://github.com/apostrophecms/apostrophe-svg-sprites/raw/1.0.0/demo.gif)
+
+This bundle provides a piece subclass that manages and renders SVG sprites referenced from external maps and an accompanying widget for display.
+
+This bundle assumes you are compiling/managing the SVG maps on your own (Gulp, Webpack, manual) and leaving them in a publicly accessible place (`/public` for example).
+
+This module is *not* an interface for uploading SVG files or pasting SVG markup.
+
+Bundle consists of 
+
+* `apostrophe-svg-sprites` as a piece subclass for managing SVGs, similar to `apostrophe-images`
+* `apostrophe-svg-sprites-widgets` as a way to render the SVG on a page
+
+`apostrophe-svg-sprites` supports multiple source maps for multiple sets of SVGs.
+
+## Example configuration
+
+```javascript
+// in app.js
+// We must declare the bundle!
+bundles: [ 'apostrophe-svg-sprites' ],
+modules: {
+  'apostrophe-svg-sprites': {
+    maps: [
+      {
+        label: 'Social Media Icons',
+        name: 'social',
+        file: 'svg/social.svg'
+      },
+      {
+        label: 'Places Icons',
+        name: 'places',
+        file: 'svg/places.svg'
+      }
+    ]
+  },
+  'apostrophe-svg-sprites-widgets': {},
+}
+
+
+```
+## Example markup output
+
+```HTML
+  <svg>
+    <use xlink:href="{{ svg.map }}#{{ svg.id }}"></use>
```

Comment:
Where did svg come from in this snippet? Work it from `data` down to how you got this...

Reply:
Or from the widget... or... not sure what's going on actually (: